### DISCUSSION TITLE
nuclei-templates not found

### DISCUSSION THREAD

**Discussion by abusabiha** [projectdiscovery/nuclei#2573] [Q&A]
category:Q&A Why Nuclei is showing nuclei-templates not found: version string empty? Even I can't updaate-templates
![Screenshot_2022-09-06_20_52_10](https://user-images.githubusercontent.com/86252592/188764994-431d53da-bdc7-41b9-bb41-dc643dfb9c23.png)
![Screenshot_2022-09-06_20_43_32](https://user-images.githubusercontent.com/86252592/188765043-23e19deb-2924-4552-8706-2d0b2347f7d2.png)
![Screenshot_2022-09-06_20_43_07](https://user-images.githubusercontent.com/86252592/188765052-257945c5-b156-4e9a-a107-f0c0a9731cc6.png)

**Comment by ehsandeep** [comment]
@abusabiha can you share the output of `nuclei -hc`
